backend/src/main/java/easy/gc_coffee_api/dto/MenuResponseDto.java

```diff
@@ -9,16 +11,40 @@
 @Getter
 @NoArgsConstructor(access = AccessLevel.PROTECTED)
 public class MenuResponseDto {
+    private Long id;
     private String name;
     private int price;
     private Category category;
+    private Long fileId;
     private String thumbnailUrl;
 
     @Builder
-    public MenuResponseDto(String name, int price, Category category, String thumbnailUrl) {
+    public MenuResponseDto(Long id, String name, int price, Category category, String thumbnailUrl, Long fileId) {
+        this.id = id;
         this.name = name;
         this.price = price;
         this.category = category;
+        this.fileId = fileId;
         this.thumbnailUrl = thumbnailUrl;
     }
+    public MenuResponseDto(Long id, String name, int price, Category category, File file){
+        this(id, name, price, category, getUrl(file), getFileId(file));
+    }
+
+    private static String getUrl(File file) {
+        if (file == null) {
+            return null;
+        }
+        // 임의로 지정하였습니다. -> 추후 수정 필요합니다.
+        String prefix = "http://localhost:8084/";
```

Comment:
```suggestion
        @Value("${app.host}")
        private String host;
```
이렇게 yaml에 등록해서 사용하면 host가 노출되지 않아서 좋은 방법입니다. 나중에 cicd할 때는 yaml을 빌드에서 제외시키면 돼요.

Reply:
감사합니다. 배웠었는데, 까먹었네요. 감사합니다.!